src/System.Diagnostics.DiagnosticSource/src/System.Diagnostics.DiagnosticSource.csproj

```diff
@@ -50,14 +43,14 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
     <NoWarn>0420</NoWarn>
   </PropertyGroup>
-
```

Comment:
Did you intend to make all of these formatting changes (e.g. removing blank lines)?  I assume VS just did that automatically?